### ISSUE TITLE
version 2.13.3 says 2.13.1

### ISSUE THREAD

**Issue by YOwatari** [direnv/direnv#317]
I get direnv v2.13.3 from github releases.

```
$ curl -L -fsS --retry 2 -o direnv https://github.com/direnv/direnv/releases/download/v2.13.3/direnv.darwin-amd64
$ chmod +x direnv
$ ./direnv version
2.13.1
```

So,  `version.go` didnt update version number.

https://github.com/direnv/direnv/blob/f73ba7bfd916730835ca5ff507679d085e7acfb5/version.go#L3

**Comment by YOwatari** [comment]
In version 2.13.2, it also says the same.

**Comment by yermulnik** [comment]
I've got `direnv` v.2.13.3 built locally and it's fine:
```
> direnv version
2.13.3
```

By the way, just noticed that `version` argument is not documented in `direnv help` output.

**Comment by zimbatm** [comment]
thanks, I forgot to bump the version.go in the repo before doing the releases. 85a9bd9289937ace497e9986b26fb10de8925197 makes the command public now.

**Comment by yermulnik** [comment]
> 85a9bd9 makes the command public now.

cool. thanks!

**Comment by zimbatm** [comment]
should be fixed in 2.14.0. Thanks for the report!
